src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Button from "../../../inline/Button";
+import InputText from "../../../inline/InputText";
+
+import s from "./SubscriptionBlock.module.css";
+
+const SubscriptionBlock = ({ placeholderText, type }) => {
```

Comment:
I'm not sure it makes sense to make the `type` a prop. Can we just always set it to `email` in this component?

Reply:
ok